teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# teacher.rb
+class Teacher < Person
+  def initialize(_id, age, specialization, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] There is no use for `_id` here since it is not being used anywhere. Let's try to avoid unnecessary inputs and remove the `_id` as instructed previously.

  ```
  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
  ```

Reply:
Removed